src/hooks/useEncounterRole.tsx

```diff
@@ -0,0 +1,15 @@
+import { OpenmrsResource, openmrsFetch } from '@openmrs/esm-framework';
+import useSWRImmutable from 'swr/immutable';
+
+export function useEncounterRole() {
+  const { data, error, isLoading } = useSWRImmutable<{ data: { results: Array<OpenmrsResource> } }, Error>(
+    '/ws/rest/v1/encounterrole?v=full',
```

Comment:
Since we aren't doing much with the encounter role, do we need the full representation?